modules/src/player.js

```diff
@@ -160,6 +160,21 @@ $(function() {
         }).done(function(data) {
           resolve(data);
         }).fail(function(jqXHR, textStatus, errorThrown) {
+          if (
+              typeof window[`onDataError_${widget.widgetId}`] === 'function'
+          ) {
+            const onDataError = window[
+              `onDataError_${widget.widgetId}`
+            ](jqXHR.status, jqXHR.responseJSON);
+
+            if (onDataError == false) {
+              xiboIC.reportFault({
```

Comment:
I think you want to do this bit if onDataError is false, but also if it is not defined at all (i.e. this is the default behaviour if the request for data fails)

Reply:
Right. Done updating the PR.